src/server.js

```diff
@@ -139,35 +140,37 @@ class Server extends EventTarget {
       websockets = networkBridge.websocketsLookup(this.url);
     }
 
-    if (typeof options !== 'object' || arguments.length > 3) {
-      data = Array.prototype.slice.call(arguments, 1, arguments.length);
-      data = data.map(item => normalizeSendData(item));
-    } else {
-      data = normalizeSendData(data);
-    }
-
-    websockets.forEach(socket => {
-      if (Array.isArray(data)) {
+    const nonNormalizedData = typeof options !== 'object' || arguments.length > 3
+      ? Array.prototype.slice.call(arguments, 1, arguments.length)
+      : data;
+    const normalizedData = typeof options !== 'object' || arguments.length > 3
```

Comment:
you have two identical ternary conditions and slice extraction. You can create another variable for the condition and remove the duplicate Array calls. Or maybe you can use the usual if/else construction:
```
let normalizedData;

if (typeof options !== 'object' || arguments.length > 3) {
  data = Array.prototype.slice.call(arguments, 1, arguments.length);
  normalizedData = data.map(item => normalizeSendData(item));
} else {
  normalizedData = normalizeSendData(data);
}
```